src/ape/managers/chain.py

```diff
@@ -438,3 +438,15 @@ def _get_test_provider(self, method_name: str) -> TestProviderAPI:
             )
 
         return provider
+
+    def mine(self, num_blocks: int = 1, timestamp: Optional[int] = None) -> str:
+        if timestamp:
+            self.pending_timestamp = timestamp
+        self.provider.mine(num_blocks)
+        return f'Mining: {num_blocks}, {timestamp}'
```

Comment:
I am a little confused by this return type. Should we just return the new head instead? (`BlockAPI`)

Reply:
Or maybe we don't need a return type here?

Reply:
Yeah agreed, no need to have a return type here, or print out anything for display.

Maybe debug log might be useful... :thinking: @unparalleled-js thoughts?

Reply:
A debug log could be useful but hard to say. Does not hurt though, it is already a noisy stream atm.